src/resolvers/index.ts

```diff
@@ -38,7 +39,23 @@ const resolvers = {
     tags: userTags,
   },
   Item: {
-    savedItem,
+    savedItem: async (item: Item, args, context: IContext) => {
+      const save = await context.dataLoaders.savedItemsByUrl.load(
+        item.givenUrl
+      );
+      if (save == null) {
+        throw new NotFoundError(`No Save found for url=${item.givenUrl}`);
```

Comment:
Should we return NotFoundError here or just null?

Reply:
It's kind of inconsistent. We do have a contract to return NotFoundError for specific id queries e.g. `savedItemById`. Good call to ask the clients... I would certainly hope that they would not just error out if there's data + errors haha

Reply:
https://pocket.slack.com/archives/C01VDFM30J2/p1670263011027919

This was uncovered during some QA testing and isn't currently included in client queries. I think the NOT_FOUND makes sense as additional data here, but we can revisit the contract if clients start to use it.